src/main.c

```diff
@@ -1,29 +1,177 @@
+#include <SDL2/SDL.h>
+#include <SDL2/SDL_events.h>
+#include <SDL2/SDL_keycode.h>
+#include <stdio.h> 
 #include <stdlib.h>
 #include <string.h>
 
 #include "bmp.h"
 #include "fractal.h"
 
-typedef unsigned long long PIXEL;
- 
+#define COLUMNS 1080
+#define ROWS 1080
+
+#define MOTION 0.06;
+
 int main() {
-	PIXEL h = 4000;
-	PIXEL w = 4000;
-	PIXEL datasize = h * w * 3;
-	char *bmp = malloc(datasize);
-	
+	// Init
+	size_t datasize = ROWS * COLUMNS * 4;
+	char *frame = malloc(datasize);
 	// Quit if memory is not available
-	if (bmp == NULL){
+	if (frame == NULL){
 		return 1;
 	}
 	
 	struct world_coordinates world;
+	world.set_left = -2.5;
+	world.set_right = 1.5;
+	world.set_down = -2.0;
+	world.set_up = 2.0;
+
+	// User's instructions
+	printf("\n\n\n");

```

Comment:
You should move this user welcome code into a function. Good names could be `print_user_welcome` or something along those lines.